cli.js

```diff
@@ -56,10 +74,44 @@ function run(file) {
 			return;
 		}
 
```

Comment:
So you are saying `fork.js` should return a Promise? But it also needs to emit `test` event for each test.